python/whylogs/api/whylabs/session/config.py

```diff
@@ -29,19 +47,48 @@ class ConfigVariableName(Enum):
     SESSION_ID = "session_id"
     WHYLABS_API_ENDPOINT = "whylabs_api_endpoint"
     USER_GUID = "user_guid"
+    DEFAULT_DATASET_ID = "default_dataset_id"
+    WHYLABS_REFERENCE_PROFILE_NAME = "whylabs_reference_profile_name"
+    WHYLABS_PRIVATE_API_ENDPOINT = "whylabs_private_api_endpoint"
+    WHYLABS_PRIVATE_S3_ENDPOINT = "whylabs_private_s3_endpoint"
+
+
+@dataclass
+class InitConfig:
```

Comment:
can we add a documentation here about how these variables interact?

Reply:
The documentation is on the `init` function in `session.py`. That's the interface that people will actually see. This thing is just an intermediate object passed into the config based on the arguments to `init`. No one will ever actually see this config type. I have a fallback list in the `init` docs so people know what to expect.